Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -588,7 +588,14 @@ private void SendLOOC(EntityUid source, ICommonSession player, string message, b
 
     private void SendDeadChat(EntityUid source, ICommonSession player, string message, bool hideChat)
     {
-        var clients = GetDeadChatClients();
+        var dead_clients = GetDeadChatClients();
+        var clients = dead_clients.Concat(
+            GetRecipients(source, VoiceRange)
+            .Select(p => p.Key)
+            .Where(p => HasComp<DeadHearingComponent>(p.AttachedEntity))
+            .Select(p => p.Channel)
+            .Except(dead_clients)); // Remove duplicates
```

Comment:
Please don't abuse LINQ like this it's incredibly slow.

Reply:
![linq](https://github.com/space-wizards/space-station-14/assets/39013340/e8306513-bd2e-4ccb-8d9d-32a9881b170d)